decrypt_creds.py

```diff
@@ -69,9 +78,7 @@ def get_credentials():
     "TRAVIS_PUBLIC_USER": "GitHub ID of GitHub account with access to Travis Public",
     "TRAVIS_PUBLIC_ID": "The ID of the Travis user. Use the Travis API to get this (for Public)",
     "TRAVIS_PUBLIC_TOKEN": Use the Travis API to get the Travis token (for the Travis Public account)",
-    "DUO-HOST": "xxxxxxxx.duosecurity.com",
-    "DUO-IKEY": "The IKEY for Duo",
-    "DUO-SKEY": "The SKEY for Duo"
+    "DUO_YOUR_DOMAIN": "your-domain-here.com,xxxxxxxx.duosecurity.com,THEDUOIKEY,THEDUOSKEY"
```

Comment:
I think I would prefer to keep this as is -- it won't break users who upgrade.

Unless you can convince me that this way is better.